Tests/MapboxMapsTests/MigrationGuide/MigrationGuideIntegrationTests.swift

```diff
@@ -202,7 +202,11 @@ class MigrationGuideIntegrationTests: IntegrationTestCase {
             let customHTTPService = CustomHttpService()
 
             func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-                HttpServiceFactory.setUserDefinedForCustom(customHTTPService)
+                // commenting this out for testing purposes to avoid
+                // interfering with global state that may impact other
+                // tests
+                //
+                // HttpServiceFactory.setUserDefinedForCustom(customHTTPService)
```

Comment:
Is this meant to merge or was this for some experimentation

Reply:
Meant to merge. I will open a ticket to follow up in a better way later.